include/ccf/app_interface.h

```diff
@@ -27,5 +30,11 @@ namespace ccfapp
    */
   std::shared_ptr<ccf::RpcFrontend> get_rpc_handler(
     ccf::NetworkTables& network, AbstractNodeContext& context);
-  // SNIPPET_END: rpc_handler
+
+  /** To be implemented by the application to be registered by CCF.
```

Comment:
I think this comment is a little confusing: C++ apps don't need to implement this and only the `js_generic` app (which is so far maintained by us) needs to do that.

Reply:
Relatedly, `js_plugin.h` is included here but the app may be C++ only. Is there a way `get_js_plugins()` could be separated from the generic app interface?

Reply:
> I think this comment is a little confusing: C++ apps don't need to implement this and only the `js_generic` app (which is so far maintained by us) needs to do that.

C++ apps don't need to, but they can, and if they do then it affects JS governance.

Reply:
"affects JS governance": Good point, I'd missed that.